### PR TITLE
Notify user about workspace 'stopped event' if previous state was 'starting.'

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13042

### What does this PR do?
Workspace loader: notify user about workspace 'stopped event' if previous state was 'starting.': show error message, hide progress animation, display reload link.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### CODE REVIEW

**Comment by musienko-maxim** [comment]
Your build is failure
[Build details](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/CHE-7-happy-path-pull-request-builder/1/)
[Test report](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/CHE-7-happy-path-pull-request-builder/1/)

File: workspace-loader/src/workspace-loader.ts [hunk 1]
```diff
@@ -208,12 +208,17 @@ export class WorkspaceLoader {
...
             masterApi.subscribeWorkspaceStatus(this.workspace.id,
-                (message: any) => {
+                (message: WorkspaceStatusChangedEvent) => {
                     if (message.error) {
                         reject(new Error(`Failed to run the workspace: "${message.error}"`));
                     } else if (message.status === 'RUNNING') {
                         this.checkWorkspaceRuntime().then(resolve, reject);
                     } else if (message.status === 'STOPPED') {
+                        if (message.prevStatus == 'STARTING') {
+                            this.loader.error('Workspace unexpected stopped during starting phase.');
```
Comment (olexii4) [inline workspace-loader/src/workspace-loader.ts]: I think "Workspace unexpected stopped" is enough.
WDYT?

Comment (AndrienkoAleksandr) [reply workspace-loader/src/workspace-loader.ts]: I simplified error message.

**Comment by AndrienkoAleksandr** [comment]
ci-build

**Comment by AndrienkoAleksandr** [comment]
ci-test

**Review by mmorhun — APPROVED** [review]

**Review by olexii4 — APPROVED** [review]

**Comment by artaleks9** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1741/) doesn't show any regression against this Pull Request.

**Pull request merged**